Remove dependency to eigenpy, to avoid a dependency on a python version

diff --git CMakeLists.txt CMakeLists.txt
index a3d9831..8c2d4e2 100644
--- CMakeLists.txt
+++ CMakeLists.txt
@@ -41,7 +41,6 @@ SET(PKG_CONFIG_ADDITIONAL_VARIABLES
 
 SETUP_PROJECT()
 # Search for dependencies.
-ADD_REQUIRED_DEPENDENCY("eigenpy")
 ADD_REQUIRED_DEPENDENCY("pinocchio >= 1.2.6")
 ADD_REQUIRED_DEPENDENCY("dynamic-graph >= 3.0.0")
 ADD_REQUIRED_DEPENDENCY("sot-core >= 3.0.0")
diff --git src/CMakeLists.txt src/CMakeLists.txt
index c8df282..dc663fa 100644
--- src/CMakeLists.txt
+++ src/CMakeLists.txt
@@ -43,7 +43,6 @@ FOREACH(lib ${plugins})
   TARGET_LINK_LIBRARIES(${libname} ${Boost_LIBRARIES})
 
   PKG_CONFIG_USE_DEPENDENCY(${libname} pinocchio)
-  PKG_CONFIG_USE_DEPENDENCY(${libname} eigenpy)
   PKG_CONFIG_USE_DEPENDENCY(${libname} sot-core)
   PKG_CONFIG_USE_DEPENDENCY(${libname} dynamic-graph)
 
